src/libraries/Common/src/Interop/Windows/Interop.Libraries.cs

```diff
@@ -19,7 +19,7 @@ internal static partial class Libraries
         internal const string Odbc32 = "odbc32.dll";
         internal const string Ole32 = "ole32.dll";
         internal const string OleAut32 = "oleaut32.dll";
-        internal const string PerfCounter = "perfcounter.dll";
+        internal const string Pdh = "pdh.dll";
```

Comment:
>pdh [](start = 37, length = 3)

just wondering, does pdh.dll exist on all Windows skus we support? I guess so but wanted to ensure we looked at that.

Reply:
Yes.  It was already being used by perfcounter.dll so it will exist everywhere this worked before.  It's been present in Windows since XP/2k3.  It's also part of nano and server core.

Reply:
perfcounter.dll (which it was using before) imports PdhFormatFromRaw value, which MSDN says back to Windows XP, eg
https://docs.microsoft.com/en-us/windows/win32/api/pdh/nf-pdh-pdhformatfromrawvalue

As for SKU's, the tests exclude only Nano server so it apparently is available on Server Core.

Reply:
Having said that, the old code presumably failed on Nano here
```
throw new Win32Exception(Marshal.GetLastWin32Error());
```
and the new code presumably fails with EntrypointNotFoundException() ? Not sure that is important but just pointing it out.

Reply:
Oh, our comments crossed. Why do the tests exclude nano?

Reply:
I guess Nano not carrying the full framework which means the PerfCounter.dll is absent there. but maybe after this change we can try to enable the test there and look if it will succeed.

Reply:
Ah - of course. Well, that would be a happy result if it worked.

Reply:
My change did enable many test cases which read perf counters on nano.  See many of the test cases which removed conditions completely.  These tests are running and passing on nano.

I still conditioned a few tests which failed to write to perfcounters.  I can't recall if I tried removing the write condition for nano or not.  I can try this in master and see if we can enable more tests.